packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -85,7 +79,7 @@ export const OutboundShipmentListViewComponent: FC = () => {
 
       <DataTable
         pagination={{ ...pagination, total: data?.totalCount }}
-        onChangePage={onChangePage}
+        onChangePage={pagination.onChangePage}
```

Comment:
changed the shape of the object a little too.. I haven't found a good reason to retain the nesting & duplication